templates/eject-ts/server.ts

```diff
@@ -13,7 +13,7 @@ const app = Fastify({
 });
 
 // Register your application as a normal plugin.
-app.register(import("./app"));
+app.register(import("./src/app"));
```

Comment:
We should never reference the `src` folder in our code since it will break when the TS compiles to JS. TSC never changes paths by design and this will try to import a `.ts` in the `.js` dist folder.